src/Status/StatusCard/StatusCard.tsx

```diff
@@ -4,35 +4,35 @@ import { Box, Card, CardContent, Typography } from "@mui/material";
 
 import { StatusCardProps } from "./StatusCard.types";
 import StatusIcon from "../StatusIcon/StatusIcon";
-import { grey } from "@mui/material/colors";
 import statuses from "../statuses";
 
 // custom card component that will be used to display status information
-const card = ({ status, name }: StatusCardProps) => {
+const card = ({ status, name, title }: StatusCardProps) => {
   const {
     label: { text }
   } = statuses[status];
   return (
     <CardContent
       sx={{
-        ":last-child": { pb: 2 }
+        ":last-child": { pb: 2 },
+        px: 2.5,
+        py: "14px"
```

Comment:
why did we need to add these?

Reply:
Th Figma says py: 14px, px: 24px. So i adjusted slighty to achieve this.
![Landing Page – Figma - Google Chrome 9_9_2024 9_49_43 AM (2)](https://github.com/user-attachments/assets/d2db0df2-7d29-4c56-b426-289c0153ddf7)

Reply:
I think we're still a bit off then. This is what Chrome shows me in Storybook:

<img width="238" alt="image" src="https://github.com/user-attachments/assets/c7f2eceb-b122-4f72-bf74-62f40ba5b882">